src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -184,17 +184,17 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
 
         private async Task UninstallAsync(IRPackageViewModel package) {
             _coreShell.AssertIsOnMainThread();
-            if (_selectedTab == SelectedTab.InstalledPackages) {
-                IsLoading = true;
-            }
+            IsLoading = true;
```

Comment:
Or maybe make a method similar to AfterLoadUnload that will update active tab like AfterUninstall since uninstall affects every tab